aggregate/clean_aggregated.py

```diff
@@ -44,7 +44,7 @@ def order_PUMS_QOL_multiple_years(categories, measures, years):
         for c in categories:
             for r in races:
                 for m in measures:
-                    rv.append(f"{c}_{r}{y}{m}")
+                    rv.append(f"{c}{y}_{r}{m}")
```

Comment:
nice, changes are clear. checked and this doesn't break the car access QOL indicator that also uses this function